OOStepByStepTest/TestHuman.cs

```diff
@@ -0,0 +1,84 @@
+﻿using OOStepByStep;
+using OOStepByStep.Human;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace OOStepByStepTest
+{
+    public class TestHuman
+    {
+        [Theory]
+        [InlineData]
+        public void Should_return_msg_when_introduce()
```

Comment:
Probably when you use the InlineData, you could use name , age , as input so that you  could merge the test case.